### PR TITLE
Jenkinsfile: env.DONT_CREATE_TESTBEDS

### PR DESCRIPTION
**Pull request by jacek-izykowski**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -82,62 +83,68 @@ pipeline {
...
-                            node(label: 'ansible') {
-                                deleteDir()
-                                unstash 'ansible'
-
-                                script {
-                                    vmwareConfig = getVMwareConfig()
-                                    inventoryFilePath = "${env.WORKSPACE}/ansible/vm.${env.BUILD_ID}"
-                                    testEnvName = generateTestEnvName()
-                                    testEnvFolder = env.VC_FOLDER
+                    if(env.DONT_CREATE_TESTBEDS == null) {
```
Comment (kaweue) [inline Jenkinsfile]: Why not ```when``` ?

**Review by kaweue — CHANGES_REQUESTED** [review]
KR + PM

**Comment by jacek-izykowski** [comment]
recheck no bug

**Review by kaweue — APPROVED** [review]
KR + PM

**Review by katrybacka — APPROVED** [review]

**Pull request merged**